overlord/configstate/configcore/network_test.go

```diff
@@ -36,31 +36,22 @@ type networkSuite struct {
 	configcoreSuite
 
 	mockNetworkSysctlPath string
-	restores              []func()
 	mockSysctl            *testutil.MockCmd
 }
 
 var _ = Suite(&networkSuite{})
 
 func (s *networkSuite) SetUpTest(c *C) {
 	s.configcoreSuite.SetUpTest(c)
-	dirs.SetRootDir(c.MkDir())
-	s.restores = append(s.restores, release.MockOnClassic(false))
+	s.AddCleanup(release.MockOnClassic(false))
 
 	s.mockSysctl = testutil.MockCommand(c, "sysctl", "")
-	s.restores = append(s.restores, func() { s.mockSysctl.Restore() })
+	s.AddCleanup(func() { s.mockSysctl.Restore() })
```

Comment:
This could be simplified (because Restore() is already of type "func()" with no return):
```
-       s.AddCleanup(func() { s.mockSysctl.Restore() })
+       s.AddCleanup(s.mockSysctl.Restore)
```